core/modules/field_ui/field_ui.theme.inc

```diff
@@ -115,8 +117,11 @@ function theme_field_ui_view_modes($variables) {
 
   // Add the row for the default display mode.
   $row = array();
-  $row['label'] = $label . '<div class="description">' . $label_desc . '</div>';
-  $row['machine'] = t('default');
+  $row['label'] = theme('label_machine_name__node_display__' . $bundle_machine_name, array(
+    'label' => $label,
+    'machine_name' => t('default'),
```

Comment:
This was here before, but if it is a machine name, then should we be translating it?